src/main/java/com/google/devtools/build/lib/buildeventservice/BuildEventServiceProtoUtil.java

```diff
@@ -161,6 +161,12 @@ public PublishBuildToolEventStreamRequest publishBuildToolEventStreamRequest(
     if (projectId != null) {
       builder.setProjectId(projectId);
     }
+    switch (lifecycleEvent.getEventCase()) {
+      case BUILD_ENQUEUED:
+      case INVOCATION_ATTEMPT_STARTED:
+        builder.addAllNotificationKeywords(getKeywords());
+        break;
```

Comment:
please add an explicit `default: // fall through` here

Reply:
Done, though I assumed a `break;` would be preferred over a fall-through comment.